src/components/tiles/ListTile.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
```

Comment:
This is mostly a file rename (from Tile to ListTile) with very little changes other than the Props type import.